app/views/polls/show.html.erb

```diff
@@ -45,6 +45,6 @@
     </aside>
   </div>
   <div class="tabs-panel is-active" id="tab-comments">
-    <%= render "comments" %>
+    <%= render "polls/questions/comments" %>
```

Comment:
I didn't state it clearly in my previous comment, I was trying to say that since the partial is common to both Poll and Poll Questions, one single shared file should be used. In this case you're reusing polls/questions/comments, that's a bit strange, maybe a polls/comments in the polls questions show would be less strange until we can do a shared comments partial for all commentables

Reply:
Since it's the exact same file for both polls and polls/questions, would it be better to place the partial in the polls' views directory and render that in both cases? Or place it in a "shared" directory that both cases can use.

Reply:
```
  <div class="tabs-panel is-active" id="tab-comments">
    <%= render "shared/comments" %>
  </div>
```

Reply:
Yes, since there is about 4-5 more comments partials, and one day we'll get them all joined together, its a nice start to have a shared comments partial that will be used for now only from Polls and Polls Questions 👏  to serve as base for future refactors 👍

Reply:
Nice. Then pushed 341b2fa84 that implements the suggested feature. Thanks for the feedback.